hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/ConfigurationKeys.java

```diff
@@ -27,6 +27,14 @@
 @InterfaceAudience.Public
 @InterfaceStability.Evolving
 public final class ConfigurationKeys {
+
+  /**
+   * Config to specify if the configured account is HNS enabled or not.
+   * If this config is not set,
```

Comment:
fix newline in middle of sentence. There is more to get to max line char count

Reply:
Done